ARTIFACT_INVENTORY.md

```diff
@@ -25,15 +25,15 @@ registry/repository/image_name:version
 | Metrics | Alertmanager | quay.io/prometheus/alertmanager:v0.25.0|
 | Metrics | BusyBox | docker.io/library/busybox:1.31.1|
```

Comment:
Please change 
- "BusyBox" to "BusyBox (OpenSearch)" (for the reference in the Logging subsystem).
- "BusyBox" to "BusyBox (Grafana)" (for the reference in the Metrics subsystem).